tests/utils/index.js

```diff
@@ -1179,29 +1169,18 @@ const startServices = async () => {
   }
 };
 
-const runCommand = (command, silent) => {
-  const [cmd, ...params] = command.split(' ');
+const runCommand = (command, verbose = true, overrideEnv = false) => {
```

Comment:
It's slightly easier to read to take an object here, so instead of passing `boolean, boolean` you can pass `{ verbose: boolean, overrideEnv: boolean }`.